blog/forge-v6-release.md

```diff
@@ -0,0 +1,158 @@
+    url: 'https://github.com/georgexu99'
+    image_url: 'https://github.com/georgexu99.png?size=96'
+  - name: vertedinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: forge-v6-release-blog
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Electron Forge Team is excited to announce that Forge v6 is now available! This stable release contains many significant new features and improvements. Highlights of this release include the addition of our Webpack plugin as well as the introduction of the Forge Core and CLI APIs.
+
+## What Does Switching to Stable Mean?
+
+During our time in a beta release state, our team has been busy refactoring Forge’s code, improving its architecture, creating new features as well as ironing out bugs and issues.
+
+Now that we are done making changes, we’re ready to release a stable version of Forge!
+
+## Upgrading to Forge v6
+
+Forge will try to migrate from Electron Forge v5 automatically as much as possible, but some of it may need to be migrated manually. Importing your settings can be done with the same process as importing an existing Electron app to Forge:
+
+<Tabs>
+   <TabItem value="Yarn" label="Yarn" default>
+
+```
+   cd my-app
+   yarn add --dev @electron-forge/cli
+   yarn electron-forge import
+```
+
+   </TabItem>
+   <TabItem value="NPM" label="NPM">
+
+```
+   cd my-app
+   npm install --save-dev @electron-forge/cli
+   npm exec --package=@electron-forge/cli -c "electron-forge import"
+```
+
+   </TabItem>
+</Tabs>
+
+## Introducing the Webpack Plugin
+
+This release adds webpack support to your build pipeline. Webpack is a module bundler: it takes your codebase’s dependencies and modules, creates a dependency graph, and bundles them (stitching them together into a single file) into a form that your index.html can read. Bundling increases performance by reducing the number of asset calls required.
+
+Forge v6 comes with a [webpack template] that makes use of the new `@electron-forge/plugin-webpack` module, plus some preset Webpack configuration options. Configuration instructions can be found in the [docs](https://www.electronforge.io/config/plugins/webpack).
+
+The addition of webpack support brings several webpack features to your Forge app. Notable features include but are not limited to:
+- Hot module reloading
+- DevServer support
+- Adding content security policies
+- Native module support
+
+## The Forge API Has Moved
+
+The Forge v5 API has been refactored into its own separate node module. The new refactored Forge [Core API] exposes the v5 Forge API methods as well as a number of utility functions to Node, allowing you call them from within your own code.
+
+This release also adds access to the Core API from your command line interface in the form of another [CLI API]. At a high level, this is a wrapper for the Forge Core API that is accessible from the terminal.
+
+## Breaking Changes:
```

Comment:
What is this changelog comparing to? It's not very relevant if it's from v5 since v6 was a complete rewrite

Reply:
Changelog indeed refers to v5 -> v6. Should we have a breaking changes / changelog section at all then?

Reply:
I would add the breaking changes from the final beta IMO.